queries/stardog-resultset.sparql

```diff
@@ -54,16 +54,9 @@ CONSTRUCT {
       ${textmatch}
       ${filters}
 
-      OPTIONAL {
-        ?sub <http://www.w3.org/2006/time#intervalStarts> ?startOptional .
-      }
-      BIND ( COALESCE(STR(?startOptional), "0001-01-01") AS ?start )
-      BIND ( xsd:date(CONCAT(SUBSTR(?start, 0, 10), SUBSTR("0001-01-01", STRLEN(?start) + 1))) AS ?startDate )
-      BIND ( ?startDate - xsd:date(NOW()) AS ?startDelta )
-      BIND ( COALESCE(xsd:float(STRBEFORE(STRAFTER(STR(?startDelta),"P"), "D")), 0) AS ?startScore )
-      BIND ( COALESCE(?score, ?startScore) AS ?totalScore )
+      ${orderPatterns}
     }
-    ORDER BY DESC(?totalScore)
+    ${orderBy}
```

Comment:
Here we only have placeholders for patterns and the `ORDER BY` clause.

Only one ordering is supported initially.